sdk/eventhub/azure-eventhub/stress/templates/testjob.yaml

```diff
@@ -9,8 +9,35 @@ spec:
   containers:
     - name: python-eh-stress
       image: {{ .Values.image }}
-      command: ['bash', '-c', 'python3 azure_eventhub_producer_stress.py & python3 azure_eventhub_consumer_stress_sync.py']
+     
+      {{ if eq .Stress.Scenario "identity" }}
```

Comment:
ooo super cool that you've organized this into separate tests that actually test different things!

qq: Where is .Stress.Scenario specified? Is it just something that's passed in when we deploy?

Reply:
Ha, I forgot to add that file sorry -- look at values.yaml